doc/tutorials/using_agl_as_uos.rst

```diff
@@ -0,0 +1,132 @@
+.. _using_agl_as_uos:
+
+Using AGL as the User OS
+########################
+
+This tutorial describes the steps to run Automotive Grade Linux (AGL) 
+as the User OS on ACRN hypervisor and the problems we got at current stage. 
+We hope the steps documented in this article could help us to reproduce the 
+problem much easier and provide some information for further debugging.
+The regulatory model of APL NUC we used is `NUC6CAYH 
+<https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc6cayh.html>`_
+and other platforms may work too.
+
+.. image:: images/The-overview-of-AGL-as-UOS.png
+   :align: center
+
+Introduction to AGL
+*******************
+
+Automotive Grade Linux is a collaborative open source project that is 
+bringing together automakers, suppliers and technology companies to 
+accelerate the development and adoption of a fully open software stack 
+for the connected car. With Linux at its core, AGL is developing an open 
+platform from the ground up that can serve as the de facto industry 
+standard to enable rapid development of new features and technologies.
+For more information about AGL, please visit AGL’s official website:
+https://www.automotivelinux.org/.
+
+Steps for using AGL as the UOS
+******************************
+
+#. Download the release of AGL from https://download.automotivelinux.org/AGL/release/eel/, 
+   and we will use ``eel_5.1.0`` release for example.
+    
+   .. code-block:: none 
+   
+      $ cd ~
+      $ wget https://download.automotivelinux.org/AGL/release/eel/5.1.0/intel-corei7-64/deploy/images/intel-corei7-64/agl-demo-platform-crosssdk-intel-corei7-64.wic.xz
+      $ unxz agl-demo-platform-crosssdk-intel-corei7-64.wic.xz      
+
+#. Download the "kernel-iot-lts2018" kernel:
+
+   .. code-block:: none
+  
+      $ sudo mkdir ~/uos-kernel-build
+      $ cd ~/uos-kernel-build
+      $ wget https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-iot-lts2018-4.19.0-27.x86_64.rpm
```

Comment:
This will *very soon* be broken (as soon as `current` moves on to the next build version)... so I suggest replacing this with a fixed URL.:

```suggestion
      $ wget https://download.clearlinux.org/releases/26700/clear/x86_64/os/Packages/linux-iot-lts2018-4.19.0-27.x86_64.rpm
```